packages/trader/src/Modules/Contract/Containers/contract.jsx

```diff
@@ -4,10 +4,12 @@ import { Redirect, withRouter } from 'react-router';
 import { CSSTransition } from 'react-transition-group';
 import { routes } from '@deriv/shared';
 import ErrorComponent from 'App/Components/Elements/Errors';
-import { Localize } from '@deriv/translations';
+import { localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 import ContractReplay from './contract-replay.jsx';
 
+const is_dialog_erros = ['GetProposalFailure', 'ContractValidationError'];
```

Comment:
```suggestion
const dialog_errors = ['GetProposalFailure', 'ContractValidationError'];
```
`is_` prefix should be used only for boolean

Reply:
@balakrishna-binary thanks for notifying, just updated it